S2VX.Game/Leaderboard/LeaderboardContainer.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Newtonsoft.Json;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osuTK.Graphics;
+using System.IO;
+
+namespace S2VX.Game.Leaderboard {
+    public class LeaderboardContainer : BasicScrollContainer {
+
+        private const string DefaultLeaderboardFileName = "leaderboard.json";
+        private TextFlowContainer NameColumn { get; set; }
+        private TextFlowContainer ScoreColumn { get; set; }
```

Comment:
Can remove `set;` if these are only getting set in the constructor

Reply:
Implemented